src/hooks/README.md

```diff
@@ -32,3 +32,6 @@ Wraps the react router `useLocation` hook to allow a function to be called on ro
 
 ### `useScript`
 Loads a script in the DOM given a source url
+
+### `useDelayedEffect`
+Runs an effect, but only 
```

Comment:
"but only..."?

Reply:
> Curious why we wait a second before showing the spinner?

Just a UX ask forrest had. no strong opinions, but it is possible that this thing finishes before 1s, depending on how long the handle reservation checks happen

Reply:
What does it look like in the "in-between" state? Also, should it be disabled to prevent multiple clicks?

Reply:
chatted "online"!